### PR TITLE
Blog post on PyWPS 4.2

### PR DESCRIPTION
**Pull request by ldesousa**
A bit long, but it is worthy spending the time to tell the story.

### CODE REVIEW

**Review by jachym — APPROVED** [review]
Nice written, thanks

File: _posts/2018-12-20-pywps-4.2.0-released.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+The organic nature of the project has meant that the goals set out by the PSC are not always met. For instance, PyWPS 4.2 still does not fully support version 2 of the WPS standard. But such is the nature of community driven projects: evolution goes along the needs of the community and not necessarily according to grand plans. There are instead other great improvements in the pipeline, like [the containerisation of processes](https://github.com/ctu-geoforall-lab-projects/dp-laza-2018/), or [the automatic storage of outputs in a database](https://wiki.osgeo.org/wiki/Database\_Output\_Storage\_for\_PyWPS). Please get involved if these developments interest you, or if you would like other improvements to be introduced.
+
+A word also regarding the WSGI applications provided as exemple services using PyWPS. [`pywps-flask`](https://github.com/geopython/pywps-flask) remains the most popular but a backlog of small issues has built up. During the holiday season developers will be lending attention to these issues. Usability issues with the containers of both `pywps-flask` and [`pywps-django`](https://github.com/jorgejesus/pywps-django) will also be addressed during this time.
```
Comment (cehbrecht) [inline _posts/2018-12-20-pywps-4.2.0-released.md]: typo: example

File: _posts/2018-12-20-pywps-4.2.0-released.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+Finally the PSC would like to thank everyone that has contributed to PyWPS these past two years: from those doing the code heavy lifting, to testers, bug reporters, documentation contributors. Every bit of contribution is precious. Thank you all for making PyWPS what is today!
```
Comment (cehbrecht) [inline _posts/2018-12-20-pywps-4.2.0-released.md]: typo: "... what [it] is today!"

File: _posts/2018-12-20-pywps-4.2.0-released.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+title: PyWPS-4.2.0 Released
+description: PyWPS-4.2.0 Released
+keywords: PyWPS, new release
+---
+
+# PyWPS-4.2.0 Released
+
+At long last, a new PyWPS release is out. A new chapter in the PyWPS 4 story is now complete and as 2018 comes to an end, it is time to look back at what happened since the major release in December of 2016.
+
+PyWPS remains fully reliant on *pro bono* contributions without any sort of commercial organisation around it. This means the project largely depends on how the community is able to contribute an any given. However, it also means there are no clients to attend or shareholders to appease, no hard deadlines or sales demos. PyWPS evolves organically, to the tune of the community's wishes.
```
Comment (cehbrecht) [inline _posts/2018-12-20-pywps-4.2.0-released.md]: ?: ... able to contribute [an any given]

File: _posts/2018-12-20-pywps-4.2.0-released.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+Relying on voluntary contributions does not mean progress is slow, much to the contrary. In recent times PyWPS has had more simultaneous contributors than at any moment before in its history. This is largely due to the major step taken with PyWPS 4 towards modern Python technologies, but also to regular contributions from the Google Summer of Code (GSoC) programme and various students thesis. During these two years the [Project Steering Committe](http://pywps.org/development/psc.html) (PSC) was expanded to six elements, and last May PyWPS [finally graduated as an OSGeo project](http://pywps.org/2018/05/30/incubation.html).
```
Comment (cehbrecht) [inline _posts/2018-12-20-pywps-4.2.0-released.md]: ?: ... PSC was expanded to six (members instead of elements?),

**Review by cehbrecht — CHANGES_REQUESTED** [review]
Sounds good :) See comments (typos, ...).

**Pull request merged**